admin/tabs/server/environment/ServerEnvModel.js

```diff
@@ -0,0 +1,62 @@
+import {GridModel} from '@xh/hoist/cmp/grid';
+import {HoistModel, managed, XH} from '@xh/hoist/core';
+import {FieldType} from '@xh/hoist/data';
+import {checkMinVersion} from '@xh/hoist/utils/js';
+import {forOwn} from 'lodash';
+
+export class ServerEnvModel extends HoistModel {
+
+    /** @member {GridModel} */
+    @managed gridModel;
+
+    get hasMinHoistCoreVersion() {
+        return checkMinVersion(XH.environmentService.get('hoistCoreVersion'), '10.1');
+    }
+
+    constructor() {
+        super();
+
+        const {STRING} = FieldType;
```

Comment:
nit:  I find this distracting as the second line in this method -- an utterly trivial thing getting in the way of the main line of code, and with caps.  

Would we consider putting it at the level of the file, say, right after the imports on line 6?